src/fsharp/CompileOptions.fsi

```diff
@@ -69,6 +69,9 @@ val GetCoreFscCompilerOptions     : TcConfigBuilder -> CompilerOptionBlock list
 val GetCoreFsiCompilerOptions     : TcConfigBuilder -> CompilerOptionBlock list
 val GetCoreServiceCompilerOptions : TcConfigBuilder -> CompilerOptionBlock list
 
+// Apply args to TcConfigBuilder and return new list of source files
+val ApplyCommandLineArgs: TcConfigBuilder * string list * string list -> string list
```

Comment:
We don't yet enforce this, and don't do it often enough in the compiler, but please add argument names to the signature, e.g. 

```fsharp
val ApplyCommandLineArgs: x: TcConfigBuilder * y: string list * z: string list -> string list
```